subprojects/packagefiles/libgpiod/bindings/cxx/tests/meson.build

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+catch2_dep = dependency('catch2')
+
+gpiod_cxx_test = executable('gpiod-cxx-test',
+                            'gpiod-cxx-test-main.cpp',
+                            'gpiod-cxx-test.cpp',
+                            'gpio-mockup.cpp',
+                            'gpio-mockup.hpp',
+                            'tests-chip.cpp',
+                            'tests-event.cpp',
+                            'tests-event.cpp',
+                            'tests-iter.cpp',
+                            'tests-line.cpp',
+                            include_directories: '.',
```

Comment:
> implicit_include_directories (since 0.42.0): a boolean telling whether Meson adds the current source and build directories to the include path, defaults to true.

It's unneeded in this case.

Reply:
Ack.